contributing/code/pull_requests.rst

```diff
@@ -147,6 +147,15 @@ work:
     for the ``5.4`` branch, the PR will also be applied by the core team on
     all the ``6.x`` branches that are still maintained.
 
+.. warning::
+
+    For a short timeframe before a new major version is released, we are in a
```

Comment:
Is it only before a major version? I have the feeling that I already saw some feature freeze for minor version to stabilize the whole thing, but I may be wrong :thinking: